api/live.js

```diff
@@ -19,6 +19,35 @@ router.get("/tasks", async (req, res) => {
   }
 });
 
+router.get("/houseInfo", async (req, res) => {
```

Comment:
We going to need some perf monitoring on this lol. Pretty much loading the entire DB

Reply:
Yeah, true, I wanna do a stress test pass on these routes once we're feature complete to make sure it can handle it